### PR TITLE
Admin urls

### PR DESCRIPTION
**Pull request by DavieReid**
Provide a few APIs to expose info about the mosaic fs and manage active sources

### CODE REVIEW

File: packages/cli/src/serve.ts [hunk 1]
```diff
@@ -22,6 +21,64 @@ export default async function serve(config, port, scope) {
...
+  app.put('/_mosaic_/source/stop', async (req, res) => {
+    res.contentType('application/text');
+    const { name } = req.body;
+    if (name) {
+      try {
+        await mosaic.stopSource(String(name));
+        res.send(`${name} stopped successfully.`);
+      } catch (e) {
+        console.error(e);
+        res.status(500).send(`${name} not stopped.  Check you have the right name!`).end();
```
Comment (joshwooding) [inline packages/cli/src/serve.ts:60]:
```suggestion
        res.status(500).send(`${name} not stopped.  Check you have the right name!`);
```

File: packages/cli/src/serve.ts [hunk 2]
```diff
@@ -22,6 +21,64 @@ export default async function serve(config, port, scope) {
...
+  app.put('/_mosaic_/source/restart', async (req, res) => {
+    res.contentType('application/text');
+    const { name } = req.body;
+    if (name) {
+      try {
+        await mosaic.restartSource(String(name));
+        res.send(`${name} restarted successfully.`);
+      } catch (e) {
+        console.error(e);
+        res.status(500).send(`${name} not restarted.  Check you have the right name!`).end();
```
Comment (joshwooding) [inline packages/cli/src/serve.ts:77]:
```suggestion
        res.status(500).send(`${name} not restarted.  Check you have the right name!`);
```

**Review by joshwooding — APPROVED** [review]

File: packages/cli/src/serve.ts (hunk 2 above)
Comment (DavieReid) [reply packages/cli/src/serve.ts:77]: Will follow up in another PR

File: packages/cli/src/serve.ts (hunk 1 above)
Comment (DavieReid) [reply packages/cli/src/serve.ts:60]: Will follow up in another PR

**Pull request merged**